test/cloudnativeproxy/proxy.go

```diff
@@ -0,0 +1,130 @@
+//go:build e2e
+
+package cloudnativeproxy
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/test/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/test/kubeobjects/daemonset"
+	"github.com/Dynatrace/dynatrace-operator/test/kubeobjects/manifests"
+	"github.com/Dynatrace/dynatrace-operator/test/kubeobjects/namespace"
+	"github.com/Dynatrace/dynatrace-operator/test/oneagent"
+	"github.com/Dynatrace/dynatrace-operator/test/operator"
+	"github.com/Dynatrace/dynatrace-operator/test/proxy"
+	"github.com/Dynatrace/dynatrace-operator/test/sampleapps"
+	"github.com/Dynatrace/dynatrace-operator/test/secrets"
+	"github.com/Dynatrace/dynatrace-operator/test/setup"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+const (
+	dynatraceNetworkPolicy       = "../testdata/network/dynatrace-denial.yaml"
+	httpsProxy                   = "https_proxy"
+	sampleNamespaceNetworkPolicy = "../testdata/network/sample-ns-denial.yaml"
+	sampleNamespace              = "test-namespace-1"
+	DtProxy                      = "DT_PROXY"
```

Comment:
```suggestion
	dtProxy                      = "DT_PROXY"
```
it's not used outside of the package so it does not have to be exported